bot.rb

```diff
@@ -0,0 +1,49 @@
+require 'telegram/bot'
+require 'dotenv/load'
+
+LIB_PATH = "lib"
+Dir.foreach(LIB_PATH){|f| require_relative File.join(LIB_PATH,f) if f =~ /.*\.rb/}
+
+def content_for_gif? bot, message
+  return MakeGifFromUrlCommand.new(bot, message, message.text) if message.text =~ %r{https?://.*\..*/?.*}
+  return MakeGifFromFileCommand.new(bot, message) if message[:video]
+  false
+end
+
+def get_general_command bot, message
+  command = message.text.downcase.to_sym if message.text
+  if COMMANDS.include? command
+    COMMANDS[command].new bot, message
+  else
+    result = content_for_gif? bot, message
+    if result
```

Comment:
Try to avoid such complex if constructions. it could be written easier:
```
result = content_for_gif?(bot, message)
return result if result
UnknownComamnd.new(bot, message)
```